### ISSUE TITLE
http-go and redis-go templates needs updated go.sum files for 2.0 release

### ISSUE THREAD

**Issue by mikkelhegn** [fermyon/spin#1996]
Using the http-go and redis-go templates with a canary build, produces the following error on `spin build`:

```
Building component temp with `tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go`
go: updates to go.mod needed; to update it:
	go mod tidy
Error: Build command for component temp failed with status Exited(1)
```

**Comment by adamreese** [comment]
I looked at this again and think I closed #1970 prematurely. I think https://github.com/fermyon/spin/blob/main/.github/gh-checktags.sh needs to be updated to support v2

**Comment by tschneidereit** [comment]
@adamreese IIUC you're saying this issue should be addressed? And if so, can you share how best to verify this?
